inloop/accounts/views.py

```diff
@@ -107,15 +86,36 @@ def get_success_url(self, user):
         return reverse("accounts:activation_complete")
 
 
+profile = ProfileView.as_view()
+
 activate = ActivationView.as_view()
 activation_complete = TemplateView.as_view(template_name="accounts/activation_complete.html")
 signup = SignupView.as_view()
 signup_closed = TemplateView.as_view(template_name="accounts/signup_closed.html")
 signup_complete = TemplateView.as_view(template_name="accounts/signup_complete.html")
 
-recover = Recover.as_view(success_url_name="accounts:recover_done")
-recover_done = RecoverDone.as_view()
-
-# use the contrib.auth form, since it supports the new PASSWORD_VALIDATORS setting
-reset = Reset.as_view(form_class=SetPasswordForm, success_url=reverse_lazy("accounts:reset_done"))
-reset_done = ResetDone.as_view()
+password_change = PasswordChangeView.as_view(
+    success_url=reverse_lazy("accounts:password_change_done"),
+    template_name="accounts/password_change_form.html"
+)
+password_change_done = PasswordChangeDoneView.as_view(
+    template_name="accounts/password_change_succeeded.html"
+)
+
+password_reset = PasswordResetView.as_view(
+    success_url=reverse_lazy("accounts:password_reset_done"),
+    subject_template_name="password_reset/recovery_email_subject.txt",
+    email_template_name="password_reset/recovery_email.txt",
+    template_name="password_reset/recovery_form.html"
+)
+password_reset_done = PasswordResetDoneView.as_view(
+    template_name="password_reset/reset_sent.html"
+)
+
+password_reset_confirm = PasswordResetConfirmView.as_view(
+    success_url=reverse_lazy("accounts:password_reset_complete"),
+    template_name="password_reset/reset.html"
+)
+password_reset_complete = PasswordResetCompleteView.as_view(
+    template_name="password_reset/recovery_done.html"
+)
```

Comment:
I would prefer to stick to the default `template_name` values that are given in https://docs.djangoproject.com/en/1.11/topics/auth/default/#all-authentication-views.

Reply:
Ok, I'll change the template names.